.gitignore

```diff
@@ -8,3 +8,4 @@ pkg
 .rbenv-gemsets
 
 /demo/*/Gemfile.lock
+/Gemfile.lock
```

Comment:
I want to keep these so I have a working version documented ... feel free to `bundle update` update to latest though